app/views/Notification.js

```diff
@@ -139,11 +153,30 @@ class NotificationScreen extends Component {
           </Text>
           {this.state.dataAvailable ? (
             <>
-              <VictoryChart height={0.35 * height} dependentAxis={true}>
-                <VictoryAxis dependentAxis={true} />
+              <VictoryChart
+                height={0.25 * height}
+                padding={{ top: 20, bottom: 50, left: 20, right: 40 }}
+                margin={0}
+                dependentAxis={true}>
+                <VictoryAxis
+                  dependentAxis={false}
+                  tickCount={14}
+                  tickFormat={t =>
+                    t == 1
+                      ? languages.t('label.notification_today')
+                      : t == 12
+                      ? languages.t('label.notification_2_weeks_ago')
```

Comment:
Can we invert this chart axis so that today is on the right?

Most users tend to expect temporal information in left to right form (for ltr languages)

Apologies, if this has already been discussed at length elsewhere.

Reply:
In my head, "Left" is most important -- today's exposure.  The right is the oldest exposure.  Happy to discuss, but let's keep it out of this PR and in the UI/UX channel.

Reply:
Sure, SGTM